cardinal/config.go

```diff
@@ -160,3 +189,52 @@ func (w *WorldConfig) setLogger() error {
 
 	return nil
 }
+
+func setupViper() {
+	once.Do(func() {
+		// Bind the command-line flags to Viper
+		pflag.String(tomlConfigFileEnvVariable, "", "path to the toml config file")
+		pflag.Parse()
+		err := viper.BindPFlags(pflag.CommandLine)
```

Comment:
nit: if the return value is just an error, you can inline this into the if block

```
if err := viper.BindPFlags(pflag.CommandLine); if err != nil {
	log.Debug().Err(err).Msg("...")
}
```

Reply:
updated